lighthouse-core/gather/gatherers/styles.js

```diff
@@ -98,8 +98,9 @@ class Styles extends Gatherer {
         }).then(content => {
           const styleHeader = this._activeStyleHeaders[sheetId];
           styleHeader.content = content.text;
-          styleHeader.parsedContent = getCSSPropsInStyleSheet(
-              parser.parse(styleHeader.content));
+          const parsedContent = parser.parse(styleHeader.content);
+          styleHeader.parsedContent = parsedContent.error ?
```

Comment:
you also mentioned in #902 that this should be handled further up the stack. Should an audit be the one evaluating whether or not the error is important?